features/artifacts.feature

```diff
@@ -0,0 +1,29 @@
+@artifacts
+Feature: Artifacts
+
+    Screwdriver allows you to view and download the artifacts generated during the build process at a later time.
+    Enabling zip compression when saving your artifacts may speed up the build.
+
+    Rules:
+    -　The user must be able to select and review all or some of the artifacts generated during the build
+
+    Background:
+        Given an existing pipeline with the workflow:
+            | job | triggers |
+            | main | ~commit |
+            | ziped | ~commit |
+        And "calvin" has admin permission to the pipeline
+
+    Scenario: Verify that artifacts have been saved
+        And "calvin" is logged in
+        When the "main" job is started
+        Then an event is created
+        And the "main" build succeeds
+        And artifacts were found in the build with the same event ID as the successful main job
+
+    Scenario: Verify that the artifacts have been saved with ziping enabled.
```

Comment:
zipping?

Reply:
Do we need to disable this test case until the feature is implemented?

Reply:
Same here.

Reply:
No, it's not.
This test should work regardless of the implementation of the SD_ZIP_ARTIFACTS function, so it should be fine to leave the test enabled.